doorstop/gui/main.py

```diff
@@ -99,9 +103,37 @@ def run(args, cwd, error):
 
     else:  # pragma: no cover (manual test)
 
-        root = tk.Tk()
+        root = widget.Tk()
         root.title("{} ({})".format(__project__, __version__))
+
+        from sys import platform as _platform
+
+        # # Load the icon
+        if _platform in ("linux", "linux2"):
+            # linux
+            from doorstop.gui import resources
+            root.tk.call('wm', 'iconphoto', root._w, tk.PhotoImage(data=resources.b64_doorstopicon_png))
+        elif _platform == "darwin":
+            # MAC OS X
+            pass  # TODO
```

Comment:
I'll create an issue for this. I have a Mac, but I'm not sure what is required of icon formats.

Reply:
Base on this [stackoverflow question](https://stackoverflow.com/questions/33134594/set-tkinter-python-application-icon-in-mac-os-x?rq=1), it seems to require the image to be in ".icns" format